lib/testsuite/index.js

```diff
@@ -709,11 +709,13 @@ class TestSuite {
 
     return this.runTestSuite();
   }
-}
 
-function isAssertionError(err) {
-  return (err instanceof AssertionError) || err.name.startsWith('AssertionError');
+  isAssertionError(err) {
```

Comment:
I don't see any reason of changing this as a method instead of leave it as it was.